core/src/main/java/be/isach/ultracosmetics/cosmetics/deatheffects/DeathEffectDragonBreath.java

```diff
@@ -0,0 +1,24 @@
+package be.isach.ultracosmetics.cosmetics.deatheffects;
+
+import be.isach.ultracosmetics.UltraCosmetics;
+import be.isach.ultracosmetics.config.SettingsManager;
+import be.isach.ultracosmetics.cosmetics.type.DeathEffectType;
+import be.isach.ultracosmetics.player.UltraPlayer;
+import org.bukkit.entity.Player;
+import org.bukkit.Effect;
+
+public class DeathEffectDragonBreath extends DeathEffect {
+
+    private final boolean silent = SettingsManager.getConfig().getBoolean(getOptionPath("Silent"));
```

Comment:
Best to remove this if it's not used here (same goes for the other death effects). You'd only need it if you wanted to have config options for the effect